sonic_platform_base/sonic_xcvr/sfp_optoe_base.py

```diff
@@ -144,7 +144,7 @@ def set_lpmode(self, lpmode):
         vendors has to implement accordingly.
         """
         api = self.get_xcvr_api()
-        return api.set_lp_mode(lpmode) if api is not None else None
+        return api.set_lpmode(lpmode) if api is not None else None
```

Comment:
The API definition for set_lpmode() doesn't match with sonic_platform_base/sfp_base.py